dango/types/src/account/margin.rs

```diff
@@ -38,3 +44,14 @@ pub enum QueryMsg {
     #[returns(HealthResponse)]
     Health {},
 }
+
+#[grug::derive(Serde)]
+pub struct LiquidationEvent {
```

Comment:
Let's just use single verbs, e.g. `Liquidate` to name the event types. This is consistent to how events are named in EVM as well as in our Hyperlane contracts.